### DISCUSSION TITLE
Using file abstract-pt.tex.

### DISCUSSION THREAD

**Discussion by AnaMachadoLuis** [joaomlourenco/novathesis#359] [Q&A] [answered]
Olá,

Neste momento na minha tese apenas tenho o resumo em inglês. Quando "descomento" no ficheiro` 4_files.tex` a linha (`\ntaddfile{abstract}[pt]{abstract-pt}` ) correspondente ao resumo em português e compilo o documento um erro aparece: 

**'Runaway argument?   ! Paragraph ended before \@swaparg was complete'.**

Gostaria de pedir ajuda para corrigir este erro de modo a poder ter também o resumo em português.

Obrigada,
Ana Luís

**Comment by joaomlourenco** [comment, marked as answer]
Hummm… create an empty file "abtract-pt2", and then `\ntaddfile{abstract}[pt]{abstract-pt2}`.  Do you have the same error?  If yes, the error is elsewhere.  If not, add stuff to the new abstract file incrementally, until you find the problem.

**Comment by AnaMachadoLuis** [comment]
Hi,

I followed the suggestion and it worked. Now I have the abstract in english and portuguse.

Thank you :)
